src/hydra_zen/_launch.py

```diff
@@ -203,57 +205,84 @@ def launch(
     'multirun/2021-10-19/17-50-07\\1',
     'multirun/2021-10-19/17-50-07\\2']
     """
+
+    if is_dataclass(config):
+        if to_dictconfig:
+            # convert Dataclass to a DictConfig
+            cfg = OmegaConf.structured(config)
+            config = OmegaConf.create(OmegaConf.to_container(cfg))
+        else:
+            if len(fields(config)) == 0:
+                raise ValueError(
+                    """There is an issue with your dataclass.  If you previously executed with a
+                `hydra/launcher` that utilizes cloudpickle (e.g., hydra-submitit-launcher), there is a known
+                issue with dataclasses (see: https://github.com/cloudpipe/cloudpickle/issues/386). You will have
+                to restart your interactive environment.  To avoid this issue you can use the option `to_dictconfig=True`."""
```

Comment:
This check will prevent empty configs from working:

```python
from hydra_zen import make_config

from dataclasses import fields

>>> fields(make_config())
()
```

This would break some of our basic mushin workflow examples

Reply:
Actually, when you get rid of this check, it would be good to add a regression test to ensure that `launch` always supports an empty config.

Reply:
You could check the length of the fields before and after the run, and then raise a warning to the user.

Reply:
yeah, I was just thinking about that. I'm not sure we can have a check for this (maybe a warning but that could get irritating).